multiwallet.py

```diff
@@ -356,6 +356,7 @@ class MyPrompt(Cmd):
     )
     intro = (
         "Welcome to multiwallet, the stateless multisig bitcoin wallet.\n"
+        "This tool is free and there is NO WAARRANTY OF ANY KIND.\n"
```

Comment:
spelling mistake in WAARRANTY

Reply:
Haha, I'm normally a decent speller but when I'm writing code I spell like a child 🤦‍♂️

Fixed.